tests/unit_tests/test_gameconnection.py

```diff
@@ -338,23 +338,11 @@ def assert_message_sent(game_connection: GameConnection, command, args):
     game_connection.abort.assert_not_called()
 
 
-async def test_handle_action_GameResult_victory_ends_sim(
+async def test_handle_action_GameEnded_ends_sim(
```

Comment:
Maybe this test should have a game with at least two players to make sure it works correctly if there are multiple connections?
(`game.check_sim_end` only does something meaningful if all connections have set `GameConnection.finished_sim`)

On second thought, this is already covered in `test_game_sim_ends_when_connections_ended_sim` of `test_gameconnection` and probably belongs there, while this test only concerns the routing.